### PR TITLE
fixes problem with editing and deleting groups

### PR DESCRIPTION
**Pull request by arysom**
and security helper is needed in CI3, now everything works 'out of the box'

### CODE REVIEW

File: application/models/adjacency_list_model.php [hunk 1]
```diff
@@ -38,6 +38,7 @@ public function __construct()
 		$this->load->database();
 		$this->load->config('adjacency_list', TRUE);
 		$this->lang->load('adjacency_list');
+		$this->load->helper('security');
```
Comment (michalsn) [inline application/models/adjacency_list_model.php]: It's not a real place for this helper - I would rather encourage people to load it automatically. For now - please delete this line. 
Afterall... we should stop using `xss_clean` on input.

File: application/models/adjacency_list_model.php [hunk 2]
```diff
@@ -285,19 +286,20 @@ public function update_group($id, $data)
 	 *
 	 * @return bool
 	 */
-	public function delete_group($id)
+	public function delete_group($item_id)
 	{
-		$this->db->trans_start();
+                $this->db->trans_start();
```
Comment (michalsn) [inline application/models/adjacency_list_model.php]: Only two tabs, no extra spaces please. 1 tab = 4 spaces.

File: application/models/adjacency_list_model.php [hunk 3]
```diff
@@ -285,19 +286,20 @@ public function update_group($id, $data)
...
 
 		$this->db->where('id', $item_id)->delete($this->tables['groups']);
 		$ar = $this->db->affected_rows();
 
+
```
Comment (michalsn) [inline application/models/adjacency_list_model.php]: Please remove extra empty line.

File: application/models/adjacency_list_model.php [hunk 4]
```diff
@@ -285,19 +286,20 @@ public function update_group($id, $data)
...
 		if ($ar)
 		{
 			$this->db->where('group_id', $item_id)->delete($this->tables['lists']);
 		}
 
-		$this->db->trans_complete();
+                $this->db->trans_complete();
```
Comment (michalsn) [inline application/models/adjacency_list_model.php]: Same here - only two tabs.

**Comment by michalsn** [comment]
Whoops... my testing environment for this lib is still on CI 2.x
I must upgrade later on.

Thank you for testing :)

**Comment by arysom** [comment]
I'm trying to be up to date so: merged, fixed, merged, maybe now :)?

**Comment by michalsn** [comment]
That's great.

**Pull request merged**